app/views/groups/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Editing group</h1>
+
+<%= render "form", user_id: @user.id, group: @group %>
```

Comment:
You're being `user_id` but your partial uses `@user` [instead](https://github.com/bryanmg/ammdk_colima/pull/9/files#diff-5fb01f4691ea689a9c7e6dd0bd221df0c464530179c43e096ff9c9238d97a4b2R1)